migration/v0.38/apply/10-trivy-operator-upgrade.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+
+    if [[ "${CK8S_CLUSTER}" =~ ^(wc|both)$ ]]; then
+      log_info "Upgrading trivy-operator"
+      kubectl_do wc delete crd sbomreports.aquasecurity.github.io
+      kubectl_do wc delete crd clustersbomreports.aquasecurity.github.io
+      helmfile_do wc -l name=trivy-operator apply
+      kubectl_do wc apply -f "${ROOT}"/helmfile.d/upstream/aquasecurity/trivy-operator/crds
+    fi
```

Comment:
We need to do a few things here:
1. Trivy-operator can be installed on both sc and wc so we need to have steps for both
2. Trivy-operator is optional so check first if enabled in the config before we do anything
3. This migration snippet should be runnable on a cluster that already runs the target version and should then skip performing CRD upgrades.
   It is enough to check and skip if the version of trivy running in the cluster is the one that we want to upgrade to.
5. Prefer to run the apply on CRDs before helmfile apply, as then we can be sure that they are up to date before trivy updates.

Then since we upgrade trivy here we can add it into the skips in `80-apply.sh` :smile: